eslint.config.mjs

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+
+import eslint from '@eslint/js';
```

Comment:
I don't see this in the devDeps, we might need to add it

Reply:
I think this is packaged with eslint but will check